tsdb/engine/tsm1/compact.go

```diff
@@ -176,6 +176,13 @@ func (c *Compactor) writeNewFiles(iter KeyIterator) ([]string, error) {
 		if err == errMaxFileExceeded {
 			files = append(files, fileName)
 			continue
+			// If the file only contained tombstoned entries, then it would be a 0 length
```

Comment:
To which block of code does this comment apply? It's ambiguous due to its placement.